Created mobile version of home page

alexandercyu opened the pull request:
NOTE: FAQ section changes have been made in the marble-mobile-about branch
NOTE: 30px spacing below h3 and h6 titles have been made in another branch.  Didn't want to duplicate the change here.

- Usage menu is hidden and usage text is below the image instead of on top.

mishaschwartz reviewed (changes requested):
Honestly, this one doesn't look like the Figma design either. 

- Text isn't the same size
- Margins are completely different
- "Explore the Technologies" section is indented weirdly

ShrutiK100 reviewed (changes requested):
The spacing format is 50px-[Heading-30px-Body-30px-Button]-50px-[Heading-30px-Body-30px-Button].
For technology card -50px-Heading-30px-[Subheading-20px-Body-20px-Button]-30px-[Subheading-20px-Body-20px-Button]-30px-[Subheading-20px-Body-20px-Button]
Notice the 24px margin on right and left side on the screen.
Margin on right side of the screen is much higher than 24px
All heading should be in the same line "About Marble" and "Getting Started" doesn't seem to be in one line.
Student , Researcher and Hobbyist section has a different format.

Please compare all components of the Figma design to align with the design.

alexandercyu commented:
> The spacing format is 50px-[Heading-30px-Body-30px-Button]-50px-[Heading-30px-Body-30px-Button]. For technology card -50px-Heading-30px-[Subheading-20px-Body-20px-Button]-30px-[Subheading-20px-Body-20px-Button]-30px-[Subheading-20px-Body-20px-Button] Notice the 24px margin on right and left side on the screen. Margin on right side of the screen is much higher than 24px All heading should be in the same line "About Marble" and "Getting Started" doesn't seem to be in one line. Student , Researcher and Hobbyist section has a different format.
> 
> Please compare all components of the Figma design to align with the design.

What about the spacing of h6 headers?  They are mostly consistent with 30px below in the desktop design and 20px below in the mobile design.  But the quickstart list has 30px in desktop and mobile and the "Explore the Services" title doesn't have any space below.  I think for consistency the quickstart list should use a different font class.  I don't think an h6 should have been used there to begin with since it's not a heading.

alexandercyu commented:
Updated buttons so regular sized buttons are visible on desktop and medium sized buttons are visible on mobile as per PR #125

mishaschwartz reviewed (changes requested):
The site no longer builds (jinja throws an error). Please fix and then request a review again.

mishaschwartz reviewed (changes requested):
This has changed:

![image](https://github.com/DACCS-Climate/marble-website/assets/4380924/77fd28fa-9e6c-4e9d-8dd7-4f14d335835c)

These sections are too close together:

![image](https://github.com/DACCS-Climate/marble-website/assets/4380924/28acb7d6-1331-4f29-9912-95048cc71db8)

The feedback button aligns weirdly at some screen widths:

![image](https://github.com/DACCS-Climate/marble-website/assets/4380924/b2096cf2-c73d-41ed-8f89-46c5222bbb3b)

alexandercyu commented:
> This has changed:
> 
> ![image](https://private-user-images.githubusercontent.com/4380924/331246772-77fd28fa-9e6c-4e9d-8dd7-4f14d335835c.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTU4OTAwOTcsIm5iZiI6MTcxNTg4OTc5NywicGF0aCI6Ii80MzgwOTI0LzMzMTI0Njc3Mi03N2ZkMjhmYS05ZTZjLTRlOWQtOGRkNy00ZjE0ZDMzNTgzNWMucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDUxNiUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDA1MTZUMjAwMzE3WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9ZmJiNDhkMGNmM2M5OThlNjM4MjQ2MWI0NDhiMTNjODlhYzhiYjBhZDIxYzdkNDhmMTE3YTY4YmM4NzY2ODQ4OSZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.ctu6FlS5FzCQvvqfyHh2c6lhfWM_9GhNt2sRwZpIbjs)
> 
> These sections are too close together:
> 
> ![image](https://private-user-images.githubusercontent.com/4380924/331246976-28acb7d6-1331-4f29-9912-95048cc71db8.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTU4OTAwOTcsIm5iZiI6MTcxNTg4OTc5NywicGF0aCI6Ii80MzgwOTI0LzMzMTI0Njk3Ni0yOGFjYjdkNi0xMzMxLTRmMjktOTkxMi05NTA0OGNjNzFkYjgucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDUxNiUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDA1MTZUMjAwMzE3WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9ODAxN2Q0ZTMxMzE5MmE5NzY2MTBkN2UwMjNiYTRmZjQ2NzE1M2M3Y2NlZjZlZjI5N2VlZDBkMTczNWRlMzAzNSZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.yNH53lIANbjAOmXajdYeD7WLSuluELackV2-w9yqQJI)
> 
> The feedback button aligns weirdly at some screen widths:
> 
> ![image](https://private-user-images.githubusercontent.com/4380924/331247453-b2096cf2-c73d-41ed-8f89-46c5222bbb3b.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTU4OTAwOTcsIm5iZiI6MTcxNTg4OTc5NywicGF0aCI6Ii80MzgwOTI0LzMzMTI0NzQ1My1iMjA5NmNmMi1jNzNkLTQxZWQtOGY4OS00NmM1MjIyYmJiM2IucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDUxNiUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDA1MTZUMjAwMzE3WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9YzU1ZDBhMGQxMTA2Zjc2OTdiMmIyNjkxMTI0NzlmYzU5ODhmZWE1NWE5N2NlMWQzMGVlZGFhNzk0ZGNiMWViNyZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.jXAF-ObaOyE5BWRwGLYwhSrqKPLHglzIguPcqqzxzdo)

Fixed the banner issues and added some margins for the spacing issues.

mishaschwartz reviewed (changes requested):
> These sections are too close together:

This still isn't fixed

And the feedback button should not move beside the text until the feedback section stacks on top of the useful links section

![image](https://github.com/DACCS-Climate/marble-website/assets/4380924/0f6bc88f-d708-4a3c-8972-e0f171a4ca3e)

alexandercyu commented:
> > These sections are too close together:
> 
> This still isn't fixed
> 
> And the feedback button should not move beside the text until the feedback section stacks on top of the useful links section
> 
> ![image](https://private-user-images.githubusercontent.com/4380924/331607457-0f6bc88f-d708-4a3c-8972-e0f171a4ca3e.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTU5NTI0NzksIm5iZiI6MTcxNTk1MjE3OSwicGF0aCI6Ii80MzgwOTI0LzMzMTYwNzQ1Ny0wZjZiYzg4Zi1kNzA4LTRhM2MtODk3Mi1lMGYxNzFhNGNhM2UucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDUxNyUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDA1MTdUMTMyMjU5WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9NDgxNjVkNGFkZjc5MmYzMTFhNjRjMzQ3MjYwYTFjMGZhODE3NDgyNzZjMjQxN2RiOTdhZjQ4NGMyNzdiODdmNyZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.gNaVeCLASgDJtF0xdCyIMuPMi3CckMfHP1i44uEGXMc)

Added spacing between the heading and the sections.

Adjusted when the feedback button shifts position.

mishaschwartz reviewed (changes requested):
It's still there

![image](https://github.com/DACCS-Climate/marble-website/assets/4380924/c31b4fca-8bf9-4d0b-87dd-94ba98db9d36)

Is it not clear what the issue is? The button is too close to the header on the next line.

This happens when the screen size is `md` I think

alexandercyu commented:
> It's still there
> 
> ![image](https://private-user-images.githubusercontent.com/4380924/331617147-c31b4fca-8bf9-4d0b-87dd-94ba98db9d36.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTU5NTQyMTQsIm5iZiI6MTcxNTk1MzkxNCwicGF0aCI6Ii80MzgwOTI0LzMzMTYxNzE0Ny1jMzFiNGZjYS04YmY5LTRkMGItODdkZC05NGJhOThkYjlkMzYucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDUxNyUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDA1MTdUMTM1MTU0WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9MTk0ZmQwNzQxZDQ5NWVmOTRlNTY2MDc1MTczM2ZjNTMxZTA1ZWU0OTY5MzRlYzFlM2IxNmY3OGQxM2U1OGQ2MiZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.7xEt02PEz1oviWK8QYenXEYzKrL808MaMM96v5C4C1A)
> 
> Is it not clear what the issue is? The button is too close to the header on the next line.
> 
> This happens when the screen size is `md` I think

I didn't see it because it only happens on one screen size.  Fixed it now.

ShrutiK100 reviewed (changes requested):
The gap between heading title body and button is all the same in technology card! I have mentioned in my previous review about the spacing format for technology card
<img width="476" alt="Screenshot 2024-05-17 at 2 16 05 PM" src="https://github.com/DACCS-Climate/marble-website/assets/161752175/b5dd8518-4c98-4a86-a885-4766fafffb24">
. Please check my previous review and the screenshot attached with this review and make the changes accordingly.

alexandercyu commented:
> The gap between heading title body and button is all the same in technology card! I have mentioned in my previous review about the spacing format for technology card <img alt="Screenshot 2024-05-17 at 2 16 05 PM" width="476" src="https://private-user-images.githubusercontent.com/161752175/331695717-b5dd8518-4c98-4a86-a885-4766fafffb24.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTU5NzA4NzksIm5iZiI6MTcxNTk3MDU3OSwicGF0aCI6Ii8xNjE3NTIxNzUvMzMxNjk1NzE3LWI1ZGQ4NTE4LTRjOTgtNGE4Ni1hODg1LTQ3NjZmYWZmZmIyNC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNTE3JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDUxN1QxODI5MzlaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1kMjg3NDE1MTQ5ZjZjNjZhYzA4M2ZjMGE2ZDVhNzhmNDU2NzhhODE4Y2IzYjdmOWFlNGQyY2Q5Yjc2NzZjZTFmJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.w0hfgzpuLfhgST2kQoLBTAcu8SNr9CrVqrqAp0ZgPl4"> . Please check my previous review and the screenshot attached with this review and make the changes accordingly.

This note at the top when the PR was created explains it.  
> NOTE: 30px spacing below h3 and h6 titles have been made in another branch. Didn't want to duplicate the change here

But I guess I will duplicate it in this branch after all.

mishaschwartz commented:
> But I guess I will duplicate it in this branch after all.

Please don't. Let's merge that other branch first and then we can look at how this affects this branch.

alexandercyu commented:
> > But I guess I will duplicate it in this branch after all.
> 
> Please don't. Let's merge that other branch first and then we can look at how this affects this branch.

Wish I'd seen this first.  I already did it.

NOTE: there is a 2.5em height applied previously when developing the desktop site to the h6 title so the content underneath it is pushed down consistently across cards.

ShrutiK100 commented:
> > The spacing format is 50px-[Heading-30px-Body-30px-Button]-50px-[Heading-30px-Body-30px-Button]. For technology card -50px-Heading-30px-[Subheading-20px-Body-20px-Button]-30px-[Subheading-20px-Body-20px-Button]-30px-[Subheading-20px-Body-20px-Button] Notice the 24px margin on right and left side on the screen. Margin on right side of the screen is much higher than 24px All heading should be in the same line "About Marble" and "Getting Started" doesn't seem to be in one line. Student , Researcher and Hobbyist section has a different format.
> > Please compare all components of the Figma design to align with the design.
> 
> What about the spacing of h6 headers? They are mostly consistent with 30px below in the desktop design and 20px below in the mobile design. But the quickstart list has 30px in desktop and mobile and the "Explore the Services" title doesn't have any space below. I think for consistency the quickstart list should use a different font class. I don't think an h6 should have been used there to begin with since it's not a heading.

The headers H6 in mobile design has 30px gap bellow in desktop and mobile both. The Quickstart menu design remains same at the browser design. And yes, we should change Quickstart menu items to Subtitle 2 instead on H6 on browser and mobile. I have made these changes in the design and added a comment.

ShrutiK100 reviewed (changes requested):
About Marble - Heading missing
<img width="261" alt="Screenshot 2024-05-22 at 1 29 48 PM" src="https://github.com/DACCS-Climate/marble-website/assets/161752175/7e76dcd4-eb29-4a8b-81f6-06f1c3926f5f">

Don't see the change in this section, needs to match the design.
<img width="270" alt="Screenshot 2024-05-22 at 1 41 12 PM" src="https://github.com/DACCS-Climate/marble-website/assets/161752175/780b44d3-ea84-42c1-b8e0-cabddbf192e6">

Menu Text drop occurring on Tablet size
<img width="401" alt="Screenshot 2024-05-22 at 2 38 31 PM" src="https://github.com/DACCS-Climate/marble-website/assets/161752175/e99941b7-d43a-4e65-a0fc-dac3a362e666">

please check if this gap is applied.
<img width="394" alt="Screenshot 2024-05-22 at 2 39 56 PM" src="https://github.com/DACCS-Climate/marble-website/assets/161752175/332be4ad-9afa-4e9c-b880-4cd9745cbbd1">

alexandercyu commented:
> About Marble - Heading missing <img alt="Screenshot 2024-05-22 at 1 29 48 PM" width="261" src="https://private-user-images.githubusercontent.com/161752175/332920910-7e76dcd4-eb29-4a8b-81f6-06f1c3926f5f.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTY0MDM5ODQsIm5iZiI6MTcxNjQwMzY4NCwicGF0aCI6Ii8xNjE3NTIxNzUvMzMyOTIwOTEwLTdlNzZkY2Q0LWViMjktNGE4Yi04MWY2LTA2ZjFjMzkyNmY1Zi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNTIyJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDUyMlQxODQ4MDRaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT00NjgxY2NmNTllMmUwZGFhZTEyMTEyNmE3NzhlOTk1ZGU4M2M0OWRiY2Y3MjZkYTlkYzlhYTQxZWVmNmE3M2EwJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.xwdu01n2dwTTnvqn4YOhjgF8RHKLaC7DwqHz-eBOoo4">
> 
> Don't see the change in this section, needs to match the design. <img alt="Screenshot 2024-05-22 at 1 41 12 PM" width="270" src="https://private-user-images.githubusercontent.com/161752175/332921216-780b44d3-ea84-42c1-b8e0-cabddbf192e6.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTY0MDM5ODQsIm5iZiI6MTcxNjQwMzY4NCwicGF0aCI6Ii8xNjE3NTIxNzUvMzMyOTIxMjE2LTc4MGI0NGQzLWVhODQtNDJjMS1iOGUwLWNhYmRkYmYxOTJlNi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNTIyJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDUyMlQxODQ4MDRaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT00YzYwNTQ5YjZmZDgyODI4Y2E1MmEyYjkwMmQyMjQ5YWRjYzI3OWUyZDM1NDM3M2NhZjc3OWY2NTUzZjAxZTI3JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.pIBbmAuxhmkZLWbbULtNah99yog-MB-97LiSVjvE9yA">
> 
> Menu Text drop occurring on Tablet size <img alt="Screenshot 2024-05-22 at 2 38 31 PM" width="401" src="https://private-user-images.githubusercontent.com/161752175/332921389-e99941b7-d43a-4e65-a0fc-dac3a362e666.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTY0MDM5ODQsIm5iZiI6MTcxNjQwMzY4NCwicGF0aCI6Ii8xNjE3NTIxNzUvMzMyOTIxMzg5LWU5OTk0MWI3LWQ0M2EtNGU2NS1hMGZjLWRhYzNhMzYyZTY2Ni5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNTIyJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDUyMlQxODQ4MDRaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0yMzI5Y2JjMTgzMWUyODUzZDM0YmVkODdjMjU3Nzc0YjVmY2UyZWU1NTYwMjFmYjQxOTI4NTE0ZGE2NTQwMzFlJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.HlUz2rND6o9WTnc1O8DDrMAWkP5cbZQsPnxsLEZuT00">
> 
> please check if this gap is applied. <img alt="Screenshot 2024-05-22 at 2 39 56 PM" width="394" src="https://private-user-images.githubusercontent.com/161752175/332922331-332be4ad-9afa-4e9c-b880-4cd9745cbbd1.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTY0MDM5ODQsIm5iZiI6MTcxNjQwMzY4NCwicGF0aCI6Ii8xNjE3NTIxNzUvMzMyOTIyMzMxLTMzMmJlNGFkLTlhZmEtNGU5Yy1iODgwLTRjZDk3NDVjYmJkMS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNTIyJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDUyMlQxODQ4MDRaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1jNDA3Y2FkOGQ1MTdiYWNmOGZiOGI5ZWIwMTQxZDllMGVhOGY4ZTYyZDYwN2NiODgzYTc3NGM0NWRiODM2OTFkJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.uyRgnX3Tx0-N9lzXLZXxHTXsLF_itIewn77JRIjnvlY">

This is likely because the conflicts have not been resolved yet.

alexandercyu commented:
> > About Marble - Heading missing <img alt="Screenshot 2024-05-22 at 1 29 48 PM" width="261" src="https://private-user-images.githubusercontent.com/161752175/332920910-7e76dcd4-eb29-4a8b-81f6-06f1c3926f5f.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTY0MDM5ODQsIm5iZiI6MTcxNjQwMzY4NCwicGF0aCI6Ii8xNjE3NTIxNzUvMzMyOTIwOTEwLTdlNzZkY2Q0LWViMjktNGE4Yi04MWY2LTA2ZjFjMzkyNmY1Zi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNTIyJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDUyMlQxODQ4MDRaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT00NjgxY2NmNTllMmUwZGFhZTEyMTEyNmE3NzhlOTk1ZGU4M2M0OWRiY2Y3MjZkYTlkYzlhYTQxZWVmNmE3M2EwJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.xwdu01n2dwTTnvqn4YOhjgF8RHKLaC7DwqHz-eBOoo4">
> > Don't see the change in this section, needs to match the design. <img alt="Screenshot 2024-05-22 at 1 41 12 PM" width="270" src="https://private-user-images.githubusercontent.com/161752175/332921216-780b44d3-ea84-42c1-b8e0-cabddbf192e6.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTY0MDM5ODQsIm5iZiI6MTcxNjQwMzY4NCwicGF0aCI6Ii8xNjE3NTIxNzUvMzMyOTIxMjE2LTc4MGI0NGQzLWVhODQtNDJjMS1iOGUwLWNhYmRkYmYxOTJlNi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNTIyJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDUyMlQxODQ4MDRaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT00YzYwNTQ5YjZmZDgyODI4Y2E1MmEyYjkwMmQyMjQ5YWRjYzI3OWUyZDM1NDM3M2NhZjc3OWY2NTUzZjAxZTI3JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.pIBbmAuxhmkZLWbbULtNah99yog-MB-97LiSVjvE9yA">
> > Menu Text drop occurring on Tablet size <img alt="Screenshot 2024-05-22 at 2 38 31 PM" width="401" src="https://private-user-images.githubusercontent.com/161752175/332921389-e99941b7-d43a-4e65-a0fc-dac3a362e666.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTY0MDM5ODQsIm5iZiI6MTcxNjQwMzY4NCwicGF0aCI6Ii8xNjE3NTIxNzUvMzMyOTIxMzg5LWU5OTk0MWI3LWQ0M2EtNGU2NS1hMGZjLWRhYzNhMzYyZTY2Ni5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNTIyJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDUyMlQxODQ4MDRaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0yMzI5Y2JjMTgzMWUyODUzZDM0YmVkODdjMjU3Nzc0YjVmY2UyZWU1NTYwMjFmYjQxOTI4NTE0ZGE2NTQwMzFlJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.HlUz2rND6o9WTnc1O8DDrMAWkP5cbZQsPnxsLEZuT00">
> > please check if this gap is applied. <img alt="Screenshot 2024-05-22 at 2 39 56 PM" width="394" src="https://private-user-images.githubusercontent.com/161752175/332922331-332be4ad-9afa-4e9c-b880-4cd9745cbbd1.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTY0MDM5ODQsIm5iZiI6MTcxNjQwMzY4NCwicGF0aCI6Ii8xNjE3NTIxNzUvMzMyOTIyMzMxLTMzMmJlNGFkLTlhZmEtNGU5Yy1iODgwLTRjZDk3NDVjYmJkMS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNTIyJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDUyMlQxODQ4MDRaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1jNDA3Y2FkOGQ1MTdiYWNmOGZiOGI5ZWIwMTQxZDllMGVhOGY4ZTYyZDYwN2NiODgzYTc3NGM0NWRiODM2OTFkJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.uyRgnX3Tx0-N9lzXLZXxHTXsLF_itIewn77JRIjnvlY">
> 
> This is likely because the conflicts have not been resolved yet.

Most of the issues above were due to merge conflicts, which have been resolved.

> > please check if this gap is applied. <img alt="Screenshot 2024-05-22 at 2 39 56 PM" width="394" src="https://private-user-images.githubusercontent.com/161752175/332922331-332be4ad-9afa-4e9c-b880-4cd9745cbbd1.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTY0MDM5ODQsIm5iZiI6MTcxNjQwMzY4NCwicGF0aCI6Ii8xNjE3NTIxNzUvMzMyOTIyMzMxLTMzMmJlNGFkLTlhZmEtNGU5Yy1iODgwLTRjZDk3NDVjYmJkMS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNTIyJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDUyMlQxODQ4MDRaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1jNDA3Y2FkOGQ1MTdiYWNmOGZiOGI5ZWIwMTQxZDllMGVhOGY4ZTYyZDYwN2NiODgzYTc3NGM0NWRiODM2OTFkJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.uyRgnX3Tx0-N9lzXLZXxHTXsLF_itIewn77JRIjnvlY">

I've added this spacing.

Maybe I missed it, but this change was made to the design last week and no notification/comment was made regarding it.

alexandercyu commented:
> > > The spacing format is 50px-[Heading-30px-Body-30px-Button]-50px-[Heading-30px-Body-30px-Button]. For technology card -50px-Heading-30px-[Subheading-20px-Body-20px-Button]-30px-[Subheading-20px-Body-20px-Button]-30px-[Subheading-20px-Body-20px-Button] Notice the 24px margin on right and left side on the screen. Margin on right side of the screen is much higher than 24px All heading should be in the same line "About Marble" and "Getting Started" doesn't seem to be in one line. Student , Researcher and Hobbyist section has a different format.
> > > Please compare all components of the Figma design to align with the design.
> > 
> > 
> > What about the spacing of h6 headers? They are mostly consistent with 30px below in the desktop design and 20px below in the mobile design. But the quickstart list has 30px in desktop and mobile and the "Explore the Services" title doesn't have any space below. I think for consistency the quickstart list should use a different font class. I don't think an h6 should have been used there to begin with since it's not a heading.
> 
> The headers H6 in mobile design has 30px gap bellow in desktop and mobile both. The Quickstart menu design remains same at the browser design. And yes, we should change Quickstart menu items to Subtitle 2 instead on H6 on browser and mobile. I have made these changes in the design and added a comment.

Changed quickstart menu items to subtitle 2 font class.

ShrutiK100 commented:
> > > About Marble - Heading missing <img alt="Screenshot 2024-05-22 at 1 29 48 PM" width="261" src="https://private-user-images.githubusercontent.com/161752175/332920910-7e76dcd4-eb29-4a8b-81f6-06f1c3926f5f.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTY0MDM5ODQsIm5iZiI6MTcxNjQwMzY4NCwicGF0aCI6Ii8xNjE3NTIxNzUvMzMyOTIwOTEwLTdlNzZkY2Q0LWViMjktNGE4Yi04MWY2LTA2ZjFjMzkyNmY1Zi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNTIyJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDUyMlQxODQ4MDRaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT00NjgxY2NmNTllMmUwZGFhZTEyMTEyNmE3NzhlOTk1ZGU4M2M0OWRiY2Y3MjZkYTlkYzlhYTQxZWVmNmE3M2EwJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.xwdu01n2dwTTnvqn4YOhjgF8RHKLaC7DwqHz-eBOoo4">
> > > Don't see the change in this section, needs to match the design. <img alt="Screenshot 2024-05-22 at 1 41 12 PM" width="270" src="https://private-user-images.githubusercontent.com/161752175/332921216-780b44d3-ea84-42c1-b8e0-cabddbf192e6.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTY0MDM5ODQsIm5iZiI6MTcxNjQwMzY4NCwicGF0aCI6Ii8xNjE3NTIxNzUvMzMyOTIxMjE2LTc4MGI0NGQzLWVhODQtNDJjMS1iOGUwLWNhYmRkYmYxOTJlNi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNTIyJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDUyMlQxODQ4MDRaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT00YzYwNTQ5YjZmZDgyODI4Y2E1MmEyYjkwMmQyMjQ5YWRjYzI3OWUyZDM1NDM3M2NhZjc3OWY2NTUzZjAxZTI3JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.pIBbmAuxhmkZLWbbULtNah99yog-MB-97LiSVjvE9yA">
> > > Menu Text drop occurring on Tablet size <img alt="Screenshot 2024-05-22 at 2 38 31 PM" width="401" src="https://private-user-images.githubusercontent.com/161752175/332921389-e99941b7-d43a-4e65-a0fc-dac3a362e666.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTY0MDM5ODQsIm5iZiI6MTcxNjQwMzY4NCwicGF0aCI6Ii8xNjE3NTIxNzUvMzMyOTIxMzg5LWU5OTk0MWI3LWQ0M2EtNGU2NS1hMGZjLWRhYzNhMzYyZTY2Ni5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNTIyJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDUyMlQxODQ4MDRaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0yMzI5Y2JjMTgzMWUyODUzZDM0YmVkODdjMjU3Nzc0YjVmY2UyZWU1NTYwMjFmYjQxOTI4NTE0ZGE2NTQwMzFlJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.HlUz2rND6o9WTnc1O8DDrMAWkP5cbZQsPnxsLEZuT00">
> > > please check if this gap is applied. <img alt="Screenshot 2024-05-22 at 2 39 56 PM" width="394" src="https://private-user-images.githubusercontent.com/161752175/332922331-332be4ad-9afa-4e9c-b880-4cd9745cbbd1.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTY0MDM5ODQsIm5iZiI6MTcxNjQwMzY4NCwicGF0aCI6Ii8xNjE3NTIxNzUvMzMyOTIyMzMxLTMzMmJlNGFkLTlhZmEtNGU5Yy1iODgwLTRjZDk3NDVjYmJkMS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNTIyJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDUyMlQxODQ4MDRaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1jNDA3Y2FkOGQ1MTdiYWNmOGZiOGI5ZWIwMTQxZDllMGVhOGY4ZTYyZDYwN2NiODgzYTc3NGM0NWRiODM2OTFkJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.uyRgnX3Tx0-N9lzXLZXxHTXsLF_itIewn77JRIjnvlY">
> > 
> > 
> > This is likely because the conflicts have not been resolved yet.
> 
> Most of the issues above were due to merge conflicts, which have been resolved.
> 
> > > please check if this gap is applied. <img alt="Screenshot 2024-05-22 at 2 39 56 PM" width="394" src="https://private-user-images.githubusercontent.com/161752175/332922331-332be4ad-9afa-4e9c-b880-4cd9745cbbd1.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTY0MDM5ODQsIm5iZiI6MTcxNjQwMzY4NCwicGF0aCI6Ii8xNjE3NTIxNzUvMzMyOTIyMzMxLTMzMmJlNGFkLTlhZmEtNGU5Yy1iODgwLTRjZDk3NDVjYmJkMS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNTIyJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDUyMlQxODQ4MDRaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1jNDA3Y2FkOGQ1MTdiYWNmOGZiOGI5ZWIwMTQxZDllMGVhOGY4ZTYyZDYwN2NiODgzYTc3NGM0NWRiODM2OTFkJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.uyRgnX3Tx0-N9lzXLZXxHTXsLF_itIewn77JRIjnvlY">
> 
> I've added this spacing.
> 
> Maybe I missed it, but this change was made to the design last week and no notification/comment was made regarding it.

Please note: Any changes made last week have been left a comment along.

ShrutiK100 reviewed (changes requested):
Not sure what are you referring to Usage menu as? 
Marble title is white in colour on mobile version, please apply standard colour scheme to the title.
<img width="474" alt="Screenshot 2024-05-27 at 5 14 03 PM" src="https://github.com/DACCS-Climate/marble-website/assets/161752175/f276e735-7005-4949-81c7-2d4564cc0449">

mishaschwartz commented on templates/partials/technology-card.html:
>>    This happens when the screen size is md I think

> I didn't see it because it only happens on one screen size. Fixed it now.

It is still not fixed, or your fixed was reverted

alexandercyu replied:
Looks good now.

alexandercyu commented:
> Not sure what are you referring to Usage menu as? Marble title is white in colour on mobile version, please apply standard colour scheme to the title. <img alt="Screenshot 2024-05-27 at 5 14 03 PM" width="474" src="https://private-user-images.githubusercontent.com/161752175/334212945-f276e735-7005-4949-81c7-2d4564cc0449.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTY5MDE4MDIsIm5iZiI6MTcxNjkwMTUwMiwicGF0aCI6Ii8xNjE3NTIxNzUvMzM0MjEyOTQ1LWYyNzZlNzM1LTcwMDUtNDk0OS04MWM3LTJkNDU2NGNjMDQ0OS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNTI4JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDUyOFQxMzA1MDJaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1kMjgwZmEyNGVlOTU0YmY5YzMzZWQ0OGNmZTQ0ZTRjNDE2ZTlmOGU4MGU0MDM3OWMxMGU5OWMwMWU5MGZlNzI1JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.LMX-CmKXa4rq5HoI4itrRsfTGpYXLjP0jgdIEgMJtw8">

This is done in the marble-mobile-node branch

alexandercyu commented on templates/partials/faq-card.html:
```diff
@@ -12,11 +12,11 @@
+        <div class="container-fluid width-faq-section">
             <div class="row">
-                <h3 class="header-title-margin"><a>FAQ</a></h3>
+                <h3 class="header-title-margin"><a>FAQ's</a></h3>
```
Shruti added an apostrophe in her  design.

ShrutiK100 replied:
I just checked, best practice is to either keep FAQ or FAQs. Alex you can change it to FAQs, I will make the change in the design and add a comment.

alexandercyu replied:
Changed the title back.

alexandercyu commented on templates/partials/usage-section.html:
```diff
@@ -1,20 +1,48 @@
-                        {% endwith %}
+<div class="d-none d-lg-flex w-100 g-0">
+    <div id="{{ id }}" class="{{ 'd-flex' if initial else 'd-none' }} w-100">
+        <div class="d-flex user-image {{ image_class }}">
```
Fixed the height of the images

mishaschwartz commented on static/css/marble.css:
```diff
@@ -642,13 +637,21 @@ h6{
 }
 
 .researcher-user-image{
+    width: 100%;
```
you forgot this for the `.student-user-image` class too

alexandercyu replied:
Might have been testing for how it looks without it, but that's been added back in.

mishaschwartz reviewed (changes requested):
Just one small change and then let's pull this in and fix anything else after the final review.

alexandercyu commented:
@ShrutiK100 I'm going to assume you've been following the messages here and are fine with the changes so far.  I'm going to merge this in and any further review will be in the final review branch.
